packages/effect/src/Cause.ts

```diff
@@ -910,7 +911,10 @@ export const isRuntimeException: (u: unknown) => u is RuntimeException = core.is
  * @since 2.0.0
  * @category errors
  */
-export const TimeoutException: new(message?: string | undefined) => TimeoutException = core.TimeoutException
+export const TimeoutException: new(
+  duration?: Duration.Duration,
```

Comment:
this is a breaking change, maybe

```ts
new(message?: string, duration?: Duration.Duration)
```

or

```ts
new(
  options?: string | {
    readonly message?: string
    readonly duration?: Duration.DurationInput
  }
)
```

Reply:
Good call @gcanti - I'll opt for the second strategy for now unless someone else has strong opposition.